src/libs/modals/modals/ModalGenericInfo.tsx

```diff
@@ -52,15 +53,15 @@ export const ModalGenericInfo: ModalFC<ModalGenericInfoData> = ({
         }}
         className={'my-16'}
       >
-        {buttonLabel}
+        {buttonLabel || t('modals.genericInfo.actionButtons.actionButton1')}
```

Comment:
note that you don't allow me to pass an empty string - becuase this check will always override it. You do it in other places too. Not a big issue - but still.